src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,15 @@
+html {
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
in my opinion, it`d be better to use classes instead of tags, and add one class for all elements you need to reset